pkg/applyinator/applyinator.go

```diff
@@ -92,13 +93,17 @@ const defaultCommand = "/run.sh"
 const cattleAgentExecutionPwdEnvKey = "CATTLE_AGENT_EXECUTION_PWD"
 const cattleAgentAttemptKey = "CATTLE_AGENT_ATTEMPT_NUMBER"
 const planRetentionPolicyCount = 64
+const restartPendingTimeout = "5m"
```

Comment:
It would be helpful to include a comment here that calls out the direct relation between this five minutes and the timeout in the `install.sh` script (via `APPLYINATOR_ACTIVE_WAIT_COUNT` and `sleep 5`)

Reply:
They're actually opposite interlocks i.e. the install sh one is checking to see if the applyinator is applying a plan, whereas the restart-pending one stops application of a plan until the restart is done.

Reply:
Ah okay, that makes sense